boards/xtensa/esp32s2/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# ESP32 board configuration
+
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ESP32S2
+	bool "ESP32S2 Development Board"
```

Comment:
Maybe change `ESP32S2` to `ESP32-S2` for the prompt so it is consistent with the documentation below.

Reply:
Sure, I tried doing that everywhere user-visible and missed this one